### DISCUSSION TITLE
The report cannot be previewed in Jupyter

### DISCUSSION THREAD

**Discussion by aleiyoo** [datapane/datapane#52] [Help] [answered]
I updated datapane to 0.9.0 and then I could not preview the report in Jupyter
`---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-3-7ae3c71b24af> in <module>
----> 1 report.preview()

~\AppData\Roaming\Python\Python38\site-packages\datapane\client\api\report\core.py in preview(self, width, height, standalone)
    283 
    284             # NOTE - iframe must be relative path
--> 285             iframe_src = self._tmp_report.relative_to(Path("..").absolute())
    286             return IFrame(src=str(iframe_src), width=width, height=height)
    287         else:

D:\anaconda3\lib\pathlib.py in relative_to(self, *other)
    902         if (root or drv) if n == 0 else cf(abs_parts[:n]) != cf(to_abs_parts):
    903             formatted = self._format_parsed_parts(to_drv, to_root, to_parts)
--> 904             raise ValueError("{!r} does not start with {!r}"
    905                              .format(str(self), str(formatted)))
    906         return self._from_parsed_parts('', root if n == 1 else '',

ValueError: 'C:\\Users\\mi\\dp-cache\\dp-tmp-lkga1svb\\dp-tmp-td5nsr44.html' does not start with 'C:\\Users\\mi\\..'`

**Comment by lanthias** [comment, marked as answer]
Thanks a lot for raising this - this is a bug and I have created an issue for you here where we can track: https://github.com/datapane/datapane/issues/57

**Reply by mands** [reply]
Bug has been fixed in https://github.com/datapane/datapane/commit/6592e1e45b26a2d86668c0a666c251e6630427c9 and will be released by eow
